dev-docs/source/Standards/AlgorithmDocumentation.rst

```diff
@@ -120,13 +120,16 @@ As the **Description** and **Usage** of an algorithm *cannot* be obtained automa
       return "Rebins data with new X bin boundaries. For EventWorkspaces, you can very quickly rebin in-place by keeping the same output name and PreserveEvents=true.";
    }
 
-``.. alias::``
```

Comment:
Are aliases no longer used?

Reply:
They're gone.

Reply:
Seems there is at least one algorithm that still uses it (and the ability to define one still exists).

https://github.com/mantidproject/mantid/blob/master/Framework/Algorithms/inc/MantidAlgorithms/ChangeBinOffset.h#L57-L58

Reply:
Interesting. I didn't find a single `.rst` file with the directive `.. alias::` in it.

Reply:
Ah, it seems aliases are handled by the `relatedalgorithms` directive now. This looks fine in that case.